packages/test/jest.test.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-env node, jest */
+
+const hrp = require("http-request-plus").default;
+const jrp = require("json-rpc-protocol");
+const _ = require("lodash/keyBy");
+
+async function call(method, params) {
+  return jrp.parse.result(
+    await hrp
+      .post("http://localhost:4000/api/", {
+        body: jrp.format.request(0, method, params),
+      })
+      .readAll("utf-8")
+  );
+}
+
+// swap promise fulfilment/rejection, very handy to test rejections
+const invertPromise = p =>
+  p.then(
+    value => {
+      throw value;
+    },
+    reason => reason
+  );
+
+function compareTimestamps(ts1, ts2) {
+  return Math.abs(ts1 - ts2) < 1e2;
+}
+
+let entries, id1, id2, id3;
+
+test("create entry", async () => {
+  id1 = await call("createEntry", { name: "name1", content: "content1" });
+  id2 = await call("createEntry", { content: "content2" });
+  id3 = await call("createEntry", { name: "name3" });
+
+  const now = Date.now();
+  const response = await call("listEntries");
+
+  expect(response).toEqual([
+    {
+      id: id1,
+      name: "name1",
+      content: "content1",
+      created: expect.any(Number),
+      updated: expect.any(Number),
+    },
+    {
+      id: id2,
+      content: "content2",
+      created: expect.any(Number),
+      updated: expect.any(Number),
+    },
+    {
+      id: id3,
+      name: "name3",
+      created: expect.any(Number),
+      updated: expect.any(Number),
+    },
+  ]);
+
+  entries = _(response, "id");
+
+  expect(entries[id1].created).toBe(entries[id1].updated);
+  expect(entries[id2].created).toBe(entries[id2].updated);
+  expect(entries[id3].created).toBe(entries[id3].updated);
+  expect(compareTimestamps(entries[id1].created, now)).toBe(true);
```

Comment:
IMHO it would be better to put the test (`expect`) itself in `compareTimestamps`.